yarn.lock

```diff
@@ -6528,6 +6528,28 @@ escodegen@^1.14.1:
   optionalDependencies:
     source-map "~0.6.1"
 
+esinstall@^0.4.0:
```

Comment:
I saw this in my PRs, too; not introduced in this PR. Is it possible that somewhere, a test is installing `esinstall` from npm? If so we’ll need to locate where this is coming from

Reply:
hm, I didn't notice this, I bet it is in there some where, and since Fred released a new version it is trying to install 0.4.0 now.

Reply:
It was the package-entrypoints test, I've updated this PR to remove it.

Reply:
Nice! Glad you found it!